ionic.config.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "myApp",
-  "app_id": "1119a57c",
+  "app_id": "3b56be40",
```

Comment:
Is this required or was it done automatically by Ionic? Might as well leave it how it was instead of polluting git history

Reply:
yeah the ionic cli changes this when pushing to ionic view. changed it back though for git sake